evm_body.yaml

```diff
@@ -653,6 +663,10 @@ eth_getTransactionReceiptsByBlock:
           description: block_number in hex OR block_hash.
           items:
             type: string
+          default:
+            [
+              '0xb3e8c898cfbf4072eaad440e8606e578a33ca4fafc27d7936d83d7392ba3e939',
+            ]
```

Comment:
if it's a type string i don’t think we want to nest it in an array. We should test all of these in a UI to double check they work

Reply:
ok sweet!